.rubocop.yml

```diff
@@ -1,10 +1,10 @@
 AllCops:
+  TargetRubyVersion: 1.9
```

Comment:
The Ruby 1.8 syntax is a thing of the past - let's embrace the new goodness Ruby allows